src/app/pages/api/openai.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenaiApiType } from '@/app/types/OpenaiApiType';
+
+const fetchData = async (): Promise<OpenaiApiType> => {
+  const response = await fetch('https://api.openai.com/v1/chat/completions', {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+      Authorization: 'Bearer ' + process.env.API_KEY,
+    },
+    body: JSON.stringify({
+      model: 'gpt-3.5-turbo',
+      messages: [{ role: 'user', content: 'Say this is a test!' }],
+      temperature: 0.7,
```

Comment:
Is this is a default temperature value?

Reply:
no specific reason for 0.7, I think this was an example from the docs, moreso to show as an example of where to pass it in

Reply:
Ok cool, we can keep a note of it for experimentation phase